resources/rafter/templates/pre-upgrade-delete-vs-job.yaml

```diff
@@ -109,3 +109,6 @@ spec:
           defaultMode: 420
           name: pre-upgrade-delete-vs-rafter-minio
         name: scripts
+    {{- if .Values.global.priorityClassName }}
```

Comment:
I think we should also add the priorityClassName to [this](https://github.com/kyma-project/kyma/blob/8eb85671fec6676e117bfa4967a16ed5b5121f54/resources/rafter/charts/controller-manager/charts/minio/templates/ensure-secret-job.yaml#L56) job in the rafter chart.